docs/output.md

```diff
@@ -110,6 +110,11 @@ The possible columns displayed by default are as follows:
 - **>= 1X** to **>= 5X** These are from Qualimap. This is the percentage of the genome covered at that particular depth coverage.
 - **% GC** This is the mean GC content in percent of all mapped reads post-deduplication. This should normally be close to the GC content of your reference genome.
 - **MT to Nuclear Ratio** This from MTtoNucRatio. This reports the number of reads aligned to a mitochondrial entry in your reference FASTA to all other entries. This will typically be high but will vary depending on tissue type.
+- **XRate** This is from Sex.DetERRmine. This is the relative coverage on the X-chromosome.
+- **YRate** This is from Sex.DetERRmine. This is the relative coverage on the Y-chromosome.
+- **Number of SNPs** This is from ANGSD. The number of SNPs left after removing sites with no data in a 5 base pair surrounding region.
+- **Contamination Estimate (Method1_ML)** This is from ANGSD. The Maximum Likelihood contamination estimate according to Method 1. The estimates using Method of Moments and/or those based on Method 2 can be unhidden through the "Configure Columns" button.
```

Comment:
```suggestion
- **Contamination Estimate (Method1_ML)** This is from the nuclear contamination function of ANGSD. The Maximum Likelihood contamination estimate according to Method 1. The estimates using Method of Moments and/or those based on Method 2 can be unhidden through the "Configure Columns" button.
```

Is there a simple way of describing what a user is looking for (i.e. high or low value) @TCLamnidis ?

Reply:
That would hinge on their specific cutoffs. Most people use 5% (`0.05`) to my knowledge, but some do change that cutoff.